configs/platforms/windows-2012r2-x64.rb

```diff
@@ -10,7 +10,7 @@
   plat.provision_with "C:/ProgramData/chocolatey/bin/choco.exe upgrade -y chocolatey --no-progress"
   plat.provision_with "C:/ProgramData/chocolatey/bin/choco.exe sources remove -name chocolatey"
 
-  plat.provision_with "C:/ProgramData/chocolatey/bin/choco.exe install -y Wix310 -version 3.10.2 -debug -x86 --no-progress"
+  plat.provision_with "C:/ProgramData/chocolatey/bin/choco.exe install -y wixtoolset -version 3.14.1 -debug --no-progress --ignore-dependencies"
```

Comment:
Not a blocking comment, just wondering why we were specifying `-x86` when building packages for `x64`. Do you know why that was?

Reply:
```suggestion
  plat.provision_with "C:/ProgramData/chocolatey/bin/choco.exe install -y wixtoolset -version 3.14.1 -debug -x86 --no-progress --ignore-dependencies"
```